### ISSUE TITLE
Publish a Romana manifest for custom network addressing ranges

### ISSUE THREAD

**Issue by danderson** [romana/romana#180]
Currently, Romana only "just works" if the Kubernetes cluster is using the default service IP range. I naively assumed that just switching any mention of a service IP in the manifest to use the correct range would be enough to fix things, but looking at http://docs.romana.io/Content/components.html , there is significant extra reconfiguration needed (configmaps, volume mounts, more flags...).

This makes it hard to test Romana in non-basic environments. It would be nicer if Romana provided a manifest with a few symbols to find/replace (e.g. ROMANA_ETCD_IP, SERVICE_CIDR...).

The context is that I'm trying to set up e2e testing for [MetalLB](https://metallb.universe.tf), and I want to test explicitly against Romana in the matrix to ensure that I remain compatible. Sadly, the complexity of setting up the cluster right now (and future complexity each time Romana updates) is a bit of a turnoff :(

**Comment by cgilmour** [comment]
Yes, the reason it works with defaults is because it makes a guess about a default config to use based on the `KUBERNETES_SERVICE_HOST` value, eg:
100.64.0.1 -> guessed as kops in AWS
10.96.0.1 -> guessed as kubeadm with default service CIDRs.
If it fails to guess correctly, you can still use a default config by setting the option on the daemon deployment, eg: adding this for kubeadm.
```
args:
- --initial-network=/var/lib/romana/initial-network/kubeadm-network.json
```

Similarly you can override the pod network CIDR using this option:
`--network-cidr-overrides=romana-network=10.x.y.z/n`
It basically overwrites the value here: https://github.com/romana/romana/blob/master/containerize/targets/daemon/kubeadm-network.json#L4-L5
